coalib/settings/ConfigurationGathering.py

```diff
@@ -283,6 +287,13 @@ def load_configuration(arg_list,
 
         sections = merge_section_dicts(sections, cli_sections)
 
+    for section_name in exclude_sections:
+        if section_name in sections:
```

Comment:
can't you maybe use `sets` / ordered sets? Then you just do some set operations: `sections - exclude_sections`

Reply:
optional though :+1:

Reply:
@Makman2  I didn't do the above change as target which is used for other purposes in the code is a list. I made the other changes you suggested.